druid-spores.md

```diff
@@ -24,22 +32,29 @@ At 3rd, 5th, 7th, and 9th level you gain access to the spells listed for that le
 
 # Halo of Spores
 
-Starting at 2nd level, you are surrounded by invisible, necrotic spores that are harmless until you unleash them on a creature nearby. When a creature you can see moves into a space within 10 feet of you or starts its turn there, you can use your reaction to deal 1d4 necrotic damage to that creature unless it succeeds on a Constitution saving throw against your spell save DC. 
+Starting at 2nd level, you are surrounded by invisible, necrotic spores that are harmless until you unleash them on a creature nearby. When a creature you can see moves into a space within 10 feet of you or starts its turn there, you can use your reaction to deal 1d4 necrotic damage to that creature. 
 
-The necrotic damage increases to 1d6 at 6th level, 1d8 at 10th level, and 1d10 at 14th level.
+The necrotic damage increases to 2d4 at 6th level, 3d4 at 10th level, and 4d4 at 14th level.
 
 # Symbiotic Entity
 
-Also at 2nd level, you gain the ability to channel magic into your spores. As an action, you can expend a use of your Wild Shape feature to awaken those spores, rather than transforming into a beast form, and you gain 4 temporary hit points for each level you have in this class. While this feature is active, you gain the following benefits:
+Also at 2nd level, you gain the ability to channel magic into your spores. As a reaction, you can expend a use of your Wild Shape feature to awaken those spores, rather than transforming into a beast form, and you gain 4 temporary hit points for each level you have in this class. While this feature is active, you gain the following benefits:
 
-* When you deal your Halo of Spores damage, roll the damage die a second time and add it to the total.
-* Your melee weapon attacks deal an extra 1d6 necrotic damage to any target they hit.
```

Comment:
Expanded symbiotic entity bonus damage to all melee attacks, not just weapon attacks.  This allows bonus damage for melee spell cantrips like *Thorn Whip* and *Primal Savagery*.